cp22/hpi-fun2/newsort/executables/generator.py

```diff
@@ -0,0 +1,55 @@
+import random
+from pathlib import Path
+
+random.seed(-6653597241841577263)
+
+
+def write_testcase(name, desc, numbers, newNumberOrdering):
+    Path(name + '.in').write_text(str(len(numbers)) + '\n' + ' '.join(map(str, numbers)) + '\n' + str(newNumberOrdering))
+    Path(name + '.desc').write_text(desc + '\n')
+
+
+write_testcase('sample1', 'Sample 1 easy', [1, 3, 5, 8], '0123456789')
+write_testcase('sample2', 'Sample 2 easy', [92, 53, 10], 9876543210)
+write_testcase('sample3', 'Sample 3', [47, 40, 99], 2164783059)
+write_testcase('sample4', 'Sample 4', [0, 40, 99], '0164783259')
+write_testcase('wrong_order', 'wrong order', [1, 2], 1234567890)
+write_testcase('same_values', 'numbers are the same', [1, 1], 1234567890)
+write_testcase('different_lengths', 'values have different lengths', [1, 11111], 1234567890)
```

Comment:
In this case, both numbers are actually 0. Is this intended?

Reply:
I think I did not clarify enough, what the number ordering string stands for. I assume you understood it this way: 
num_ordering[0] -> 1 means that in the new universe, the digit ordering replaces our 0 with a 1. 
I meant it this way:
num_ordering[0] -> 1 means that the smallest digit in this universe is 1. Therefore if we had the num_ordering 9876543210 we have 9 as the smallest digit and 0 as the biggest. And thats where I see the problem, you cant really write 0000 as a number :/ 
Do you think its then best to simply say leading zeros are allowed and its simply a complete randomization of the digits? So probably in the way you already understood above, that there is a mapping between index and digit. This could allow 0000 to be different to 00.

Reply:
I'll try to implement it that way for now :)